app/src/views/tool/ToolLog.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <view-base
+    :queries="queries" @queries-response="formatLogData"
+    ref="view" skeleton="card-info-skeleton"
+  >
+    <!-- INFO CARD -->
+    <card :title="description" icon="info-circle">
+      <b-row
+        v-for="(value, prop) in info" :key="prop"
+        no-gutters class="row-line"
+      >
+        <b-col md="3" xl="2">
+          <strong>{{ $t('logs_' + prop) }}</strong>
+        </b-col>
+
+        <b-col>
+          <span v-if="prop.endsWith('_at')">{{ value | readableDate }}</span>
+
+          <div v-else-if="prop === 'suboperations'">
+            <div v-for="operation in value" :key="operation.name">
+              <icon v-if="!operation.success" iname="times" class="text-danger" />
+              <b-link :to="{ name: 'tool-log', params: { name: operation.name } }">
+                {{ operation.description }}
+              </b-link>
+            </div>
+          </div>
+
+          <span v-else>{{ value }}</span>
+        </b-col>
+      </b-row>
+    </card>
+
+    <div v-if="info.error" class="alert alert-danger my-5">
+      <icon iname="exclamation-circle" /> {{ $t('operation_failed_explanation') }}
+    </div>
+
+    <!-- LOGS CARD -->
+    <card :title="$t('logs')" icon="file-text" no-body>
+      <template #header-buttons>
+        <b-button @click="shareLogs" variant="success">
+          <icon iname="cloud-upload" /> {{ $t('logs_share_with_yunopaste') }}
+        </b-button>
+      </template>
+
+      <b-button
+        v-if="moreLogsAvailable"
+        variant="white" class="w-100 rounded-0"
+        @click="$ref.view.fetchQueries()"
```

Comment:
```suggestion
        @click="$refs.view.fetchQueries()"
```

I have no idea what I'm doing, but it works better like this